website/docs/tools/analyzer.md

```diff
@@ -30,6 +30,7 @@ Currently, the following package managers (grouped by the programming language t
 * Java
```

Comment:
Maybe mention https://github.com/oss-review-toolkit/ort/pull/9942 or the individual commits in the commit message ?